### PR TITLE
Test Request

### PR DESCRIPTION
**Pull request by rajgane07**
This to test GA action from feature branch do not approve

### CODE REVIEW

**Comment by alex124585** [comment]
why this failed - 
QOKit test (python simulator only) / build (3.9) (push) Failing after 20s

File: .github/workflows/qokit-package.yml [hunk 1]
```diff
@@ -5,9 +5,10 @@ name: QOKit test
 
 on: # workflow_dispatch
  push:
-   branches: [ "main", "**" ]
- pull_request:
-   branches: [ "main" ]
+   #branches: [ "main", "**" ]
```
Comment (rsln-s) [inline .github/workflows/qokit-package.yml:8]: Is there a reason the lines are commented out and not simply deleted?

File: .github/workflows/qokit-python-only.yml [hunk 2]
```diff
@@ -5,9 +5,10 @@ name: QOKit test (python simulator only)
 
 on: # workflow_dispatch
  push:
-   branches: [ "main", "**" ]
- pull_request:
-   branches: [ "main" ]
+   #branches: [ "main", "**" ]
```
Comment (rsln-s) [inline .github/workflows/qokit-python-only.yml:8]: Is there a reason the lines are commented out and not simply deleted?

**Review by rsln-s — CHANGES_REQUESTED** [review]
See comments

File: .github/workflows/qokit-python-only.yml (hunk 2 above)
Comment (rajgane07) [reply .github/workflows/qokit-python-only.yml]: I commented those lines to test the GitHub Action for publishing. I have
- reverted the commented lines for 
- .github/workflows/qokit-package.yml
- .github/workflows/qokit-python-only.yml
Deleted pypi-test.yml and merged that code in pypi-test-publish.yml
Modified pypi-publish.yml to publish to pypi on release tag

I have updated codes and all actions are green

**Review by rsln-s — APPROVED** [review]
Looks great, thank you @rajgane07!

**Review by alex124585 — APPROVED** [review]

**Pull request merged**